variables.tf

```
@@ -85,3 +85,9 @@ variable "domain_enabled" {
   default     = "true"
   description = "A boolean that determines whether a DNS Zone for the kops domain is created"
 }
+
+variable "create_bucket" {
```

```suggestion
variable "bucket_enabled" {
```
We use `_enabled` flags in all our modules

---

The bucket will be enabled and used regardless of the settings. This flag selects between creating the bucket or using the existing bucket. Therefore I do not thing `enabled` is an appropriate name in this case.